interface.php

```diff
@@ -292,7 +292,7 @@ private function printPageActions($pageId) {
 
         $this->ptln('<div class="be-actions">', +2);
         $this->printAction($link, 'ttl_view', 'file-document');
-        $this->printAction($link . '&do=edit', 'ttl_edit', 'pencil');
+        $this->printAction($link . '?do=edit', 'ttl_edit', 'pencil');
```

Comment:
I guess you have 'userewrite' enabled in DW config. When ot's disabled (default) $link already contains '?'. How about:
``` php
$link . (strpos($link, '?') === FALSE ? '?' : '&') . 'do=edit'
```